chia/full_node/mempool.py

```diff
@@ -403,15 +404,30 @@ def create_bundle_from_mempool_items(
                 continue
             try:
                 cost = uint64(0 if item.npc_result.conds is None else item.npc_result.conds.cost)
-                unique_coin_spends, cost_saving, unique_additions = eligible_coin_spends.get_deduplication_info(
-                    bundle_coin_spends=item.bundle_coin_spends, max_cost=cost
-                )
+                if skipped_items >= 3:
+                    # Process this item straight away (no eligbility consideration)
+                    log.info("Processing mempool item without eligbility consideration: %s", name.hex())
+                    unique_coin_spends = []
+                    unique_additions = []
+                    for spend_data in item.bundle_coin_spends.values():
+                        unique_coin_spends.append(spend_data.coin_spend)
+                        unique_additions.extend(spend_data.additions)
+                    cost_saving = 0
```

Comment:
I would have expected that if this is a transaction eligible for identical spend deduplication, we have to just skip it. otherwise we'll include a double spend in the block, no?
alternatively, if the only reason we have to perform the expensive operation (running CLVM) is to compute `cost_savings`, we could do everything else but just over-estimate the cost of the transaction.

It's not obvious to me that this is correct. Even if it is, I think these questions ought to be answered in comments here.